### ISSUE TITLE
Staging Review finding: Implicit submit results in change of context

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#129387]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ursa Minor
**Product:** Travel Pay Claims
**Feature:** Complex Claims v1

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

With "Select date range" select under Travel claims, navigation by arrow (how keyboard only users navigate through select options) triggers an update to the page, and loads new content. This initiates a change of context that may be confusing for users. Additionally, the user has to hit [TAB] to then go back to the select to see what their selection was. It's a cumbersome interaction for users. 

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/3febfec4-66bd-48f1-be5f-82e2ba211ba7

</details>

## Recommended action

Avoid implicit submits whenever possible, as this can initiate a change in context, which is a challenge for assistive technology users. 
https://www.w3.org/WAI/WCAG21/Techniques/html/H84

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 3.2.2 On Input - Level A](https://www.w3.org/TR/WCAG22/#on-input)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by shiragoodman** [comment]
@jasonday FYI

@lakper this might be a good opportunity for ADE to assist.

**Comment by sarahhortonadhoc** [comment]
See [Slack thread](https://dsva.slack.com/archives/C05QG2H510T/p1772481904289349) and [Figma exploration](https://www.figma.com/design/NI9ziF3FsJcU9SxoABc85v/MHV-Accessible-Patterns?node-id=7335-33549&t=lZHCeB5eSNHQWT78-1).
